pkg/kv/kvserver/store.go

```diff
@@ -1952,7 +1952,6 @@ func loadFullReplicaIDsFromDisk(
 // loadAndReconcileReplicas loads the Replicas present on this
 // store. It reconciles inconsistent state and runs validation checks.
 //
-// TODO(sep-raft-log): also load *uninitialized* Replicas.
```

Comment:
We're new setting `RaftReplicaID` for all ranges including uninitialized, right? Does this count as "load uninitialized Replicas", or there is still more work to do (creating the in-memory uninit replicas)?

Reply:
We're setting it now. There still isn't a migration path for uninitialized replicas that were created before that change landed and then never touched (we do leak these routinely) but the migration path will be simply throwing away raft state belonging to such replicas.

I mentioned it here just to be sure: https://github.com/cockroachdb/cockroach/issues/93310#issuecomment-1362872896